internal/pod_models_test.go

```diff
@@ -2079,7 +2079,7 @@ var _ = Describe("pod_models", func() {
 
 		Context("with command line arguments for the sidecar", func() {
 			BeforeEach(func() {
-				cluster.Spec.Version = fdbtypes.Versions.WithCommandLineVariablesForSidecar.String()
+				cluster.Spec.Version = fdbtypes.Versions.Default.String()
```

Comment:
Do we still need a special test case for this if it is the only supported behavior?